pkg/constant/const.go

```diff
@@ -62,6 +64,12 @@ const (
 	SelectorKey = "name"
 	// FluentbitLabelKey is key for label used by fluentbit to make "release" folder
 	FluentbitLabelKey = "release"
+	// NodeServiceAccountName is a name for service account of node daemonset
+	NodeServiceAccountName = "csi-node-sa"
```

Comment:
I think we need to stop hardcoding such values and pass them as a parameters

Reply:
moved to crds